src/components/InputCode/index.tsx

```diff
@@ -8,6 +8,7 @@ import StyledInput from './styled/input';
 import getNextIndex from './helpers/getNextIndex';
 import getPrevIndex from './helpers/getPrevIndex';
 import { BaseProps } from '../types';
+import handlePasteCode from './helpers/getCodeFromClipboard';
```

Comment:
It is confusing having two names one for the file and another for the function, instead, let's use the same (getCodeFromClipboard) like:
```import getCodeFromClipboard from './helpers/getCodeFromClipboard';
Same for test where handlePasteCode is used and for the getCodeFromClipboard.js file where you ```export default handlePasteCode``` change it for ```export default getCodeFromClipboard;```